### DISCUSSION TITLE
greenPower / ZGP packets are not broadcasted by capable hue devices

### DISCUSSION THREAD

**Discussion by florianbrede-ayet** [Koenkk/zigbee2mqtt#10746] [Devices]
#### Setup
- HA with zigbee2mqtt edge on channel 11
- Conbee II as coordinator with latest firmware
- numerous hue devices (bulbs, power sockets)
- "Friends Of Hue" (PTM 215Z) switches

#### Problem
Only the coordinator is picking up and broadcasting the greenPower / ZGP packets, so only switches close to the coordinator work reliable.
None of my hue devices which are all zgp routing capable (and show up with `Endpoint 242 > Output clusters > greenPower` in z2m) are ever broadcasting ZGP messages.

I've verified this by sniffing the zigbee traffic with wireshark and a second conbee.

#### Expected Behaviour
All greenPower capable routers should broadcast messages they receive.

#### Attempts to fix
- Recommissioning the ZGP devices including channel switches
- Force removing and readding the greenPower switches from z2m

I'm a little lost at this point thinking that my hue devices were given some incorrect configuration that hinders them from broadcasting the ZGP packets.

**Comment by floyd1111** [comment]
Hi Florian,
I am currently having the same problem as you a few years ago. I am not able to route ZGP messages via any device other device than the main coordinator. Did you manage to solve it? In another issue I read, that you identified a problem with the profile the messages were sent with? Could you give me a hint to a solution?
